app/src/main/java/ch/epfl/sdp/peakar/points/LineOfSight.java

```diff
@@ -167,9 +162,9 @@ private boolean isVisible(POIPoint poiPoint, Pair<Integer, Integer> userIndexes,
         double cumulatedError = 0;
 
         if (Math.abs(pixelY) >= Math.abs(pixelX)) {
-            ratio = pixelX != 0 ? Math.abs(pixelY / pixelX) : Math.abs(pixelX);
```

Comment:
Check code climate issue : similar code blocks.
Couldn't we define a method instead ?

Reply:
I could put these similar blocks in one method, but this method would require this parameters:
* `x1`
* `x2`
* `y1`
* `y2`
* `ratio`
* `error`
* a `boolean` to indicate if the line is computed by circling on the x or y axis

I don't know it it is a good idea to have a method with this many parameters that would be 7-8 lines long